### ISSUE TITLE
[Broken] YouTube said: Unable to extract video data

### ISSUE THREAD

**Issue by sfdjo435** [blackjack4494/yt-dlc#123]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dlc version **2020.10.31**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--cookies', 'cookies.txt', '--dump-json', 'Woo-azldU1s']
[debug] Loading archive file None
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dlc version 2020.10.31
[debug] Python version 3.8.5 (CPython) - Linux-5.4.0-1026-kvm-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.3.1, ffprobe 4.3.1
[debug] Proxy map: {}
ERROR: Woo-azldU1s: YouTube said: Unable to extract video data
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/youtube_dlc/YoutubeDL.py", line 830, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.8/dist-packages/youtube_dlc/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.8/dist-packages/youtube_dlc/extractor/youtube.py", line 1903, in _real_extract
    raise ExtractorError(
youtube_dlc.utils.ExtractorError: Woo-azldU1s: YouTube said: Unable to extract video data
```

## Description

Hello, on age-gated & only playable on YouTube, videos will give the 'Unable to extract video data' error when using cookies.

When running the command without cookies: "ERROR: Video is age restricted and only playable on Youtube. Requires cookies!"

**Comment by peet1993** [comment]
This is likely to be fixed with #68. It is not part of the current release, you would have to build yourself (or run `python youtube_dlc\__main__.py <url>` inside the cloned folder) and see if that works.

**Comment by marabu88** [comment]
I confirm that such a problem exists.
If we don't use cookies we get a very fast 402 error, even if we don't abuse frequent parallel downloads.

**Comment by sfdjo435** [comment]
> This is likely to be fixed with #68. It is not part of the current release, you would have to build yourself (or run `python youtube_dlc\__main__.py <url>` inside the cloned folder) and see if that works.

Seems to be working when using cookies. When will it be released?

**Comment by pukkandan** [comment]
@sfdjo435 It has been released.
